src/other/utility/utility-en.hbs

```diff
@@ -502,15 +504,47 @@
 		<h3 class="h5"><a href="#" class="stretched-link">[Hyperlink text]</a></h3>
 		<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
 	</div>
+	<div class="col-md-4">
+		<div class="panel panel-default position-relative">
+			<div class="panel-body">
+				<img src="https://via.placeholder.com/360x203/000000/FFFFFF.png" alt="" class="img-responsive thumbnail mrgn-bttm-sm">
+				<h3 class="h5"><a href="#" class="stretched-link">[Hyperlink text]</a></h3>
+				<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+			</div>
+		</div>
+	</div>
+	<div class="col-md-4">
+		<div class="well position-relative">
+			<img src="https://via.placeholder.com/360x203/000000/FFFFFF.png" alt="" class="img-responsive thumbnail mrgn-bttm-sm">
+			<h3 class="h5"><a href="#" class="stretched-link">[Hyperlink text]</a></h3>
+			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+		</div>
+	</div>
```

Comment:
A section should be used to semantically connect the image along with its related heading.

```suggestion
	<div class="col-md-4">
		<section class="well position-relative">
			<img src="https://via.placeholder.com/360x203/000000/FFFFFF.png" alt="" class="img-responsive thumbnail mrgn-bttm-sm">
			<h3 class="h5"><a href="#" class="stretched-link">[Hyperlink text]</a></h3>
			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
		</section>
	</div>
```